src/app/redux/Login/reducers.js

```diff
@@ -1,12 +1,22 @@
-import * as Immutable from 'seamless-immutable';
+import Immutable from 'seamless-immutable';
+
+import api from '../../../config/api';
 
 import { actions } from './actions';
 
-const initialState = {};
+const initialState = {
+  token: ''
+};
 
 function reducer(state = Immutable(initialState), action) {
   switch (action.type) {
-    case actions.LOGIN:
+    case actions.USER_LOGIN_SUCCESS: {
+      api.setHeaders({ Token: action.payload.token });
```

Comment:
this code should be placed in `UserService`.
 
inside it, you shoult wait for the response, and if its OK, set the header. And then return the response